### PR TITLE
[distroless] Distroless cni simple bridge

### PR DESCRIPTION
**Pull request by universal-itengineer**
## Description

- [x] cni-simple-bridge/simple-bridge

## Why do we need it, and what problem does it solve?

This PR is a part of moving Deckhouse to distroless.
## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cni-simple-bridge
type: chore
summary:  "`simple-bridge`" use distroless based image.
impact: "`simple-bridge` pods will restart; affected clouds AWS,Azure,GCP,Yandex-Cloud."
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by universal-itengineer** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.25 6809225980 6809225980-1-con-1-25 /sys/deckhouse-oss/install:pr6469 user@34.159.211.87

**Comment by universal-itengineer** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 6809226144 6809226144-1-con-1-25 /sys/deckhouse-oss/install:pr6469 centos@3.124.239.249

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: modules/035-cni-simple-bridge/images/simple-bridge/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-binaries-artifact
+  add: /usr/lib64
+  to: /usr/lib64
+  before: setup
+  includePaths:
+  - "python3.*"
+docker:
+  ENTRYPOINT: ["/iptables-wrapper-installer.sh"]
+---
+{{ $csiBinaries := "/bin/awk /bin/chmod /bin/cat /bin/rm /bin/ln /usr/sbin/update-alternatives /bin/true /bin/echo /usr/bin/tee /bin/sleep /bin/hostname /usr/bin/python3 /usr/bin/curl /usr/bin/jq /bin/bash /bin/grep /sbin/ip /usr/sbin/bridge /usr/bin/iptables* /sbin/iptables* /sbin/ip6tables*" }}
```
Comment (RomanenkoDenys) [inline modules/035-cni-simple-bridge/images/simple-bridge/werf.inc.yaml:21]: why we need curl and chmod and tee and update-alternatives and so on ?

File: modules/035-cni-simple-bridge/images/simple-bridge/werf.inc.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  to: /lib64/iptables
+  before: setup
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-binaries-artifact
+  add: /usr/lib64
+  to: /usr/lib64
+  before: setup
+  includePaths:
+  - "python3.*"
+docker:
+  ENTRYPOINT: ["/iptables-wrapper-installer.sh"]
```
Comment (RomanenkoDenys) [inline modules/035-cni-simple-bridge/images/simple-bridge/werf.inc.yaml:19]: Please use common/iptables-wrapper

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some questions

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by universal-itengineer** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/azure'...

**Comment by universal-itengineer** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.25 6864106182 6864106182-1-con-1-25 /sys/deckhouse-oss/install:pr6469 user@34.159.60.141

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've recognized your '/e2e/abort' command and started the workflow...

  :fast_forward: `e2e destroy failed: GCP, Containerd, Kubernetes 1.25` for `refs/heads/main` [started](https://github.com/deckhouse/deckhouse/actions/runs/6866615106).

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

  :fast_forward: `e2e: Yandex.Cloud, Containerd, Kubernetes 1.25` for `deckhouse:distroless-cni-simple-bridge` [started](https://github.com/deckhouse/deckhouse/actions/runs/6866619993).


Yandex.Cloud-WithoutNAT-Containerd-1.25 - Connection string: `ssh  cloud-user@51.250.9.230`

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by universal-itengineer** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.25 6868580179 6868580179-1-con-1-25 /sys/deckhouse-oss/install:pr6469 user@34.89.204.162

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by universal-itengineer** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @universal-itengineer. I've started the workflow for label 'e2e/run/aws'...

**Review by RomanenkoDenys — APPROVED** [review]

**Review by aleksey-su — APPROVED** [review]

**Pull request merged**